lib/nodes/Func.js

```diff
@@ -101,7 +102,9 @@ class Func extends Container {
     parser.back(rightTokens);
 
     const { lastNode } = parser;
-    lastNode.isColor = colorFunctions.includes(lastNode.name.toLowerCase());
+    const lowerName = lastNode.name.toLowerCase();
+    lastNode.isColor = colorFunctions.includes(lowerName);
+    lastNode.isVar = lowerName === 'var';
```

Comment:
@shellscape So far this is a loose check, ideally it would also check the it's argument is of length 1 and has this pattern: `/--.*/`.

Reply:
done